features/step_definitions/loc_id_routing_steps.rb

```diff
@@ -24,10 +24,12 @@
 end
 
 Given(/^I visit my subject's locid$/) do
+  Capybara.current_session.driver.header('Accept', 'text/html')
```

Comment:
Could be moved to a more Central place, so it applies to all requests. Unfortunately I don't know why rack-test is even used here or where it is initialized, since our env.rb sets the default driver to poltergeist.